packages/fakta-barn-og-overfoeringsdager/package.json

```diff
@@ -21,7 +21,7 @@
     "nav-frontend-js-utils": "^1.0.9",
     "nav-frontend-knapper": "2.0.9",
     "nav-frontend-knapper-style": "0.3.35",
-    "nav-frontend-paneler": "^2.0.2",
+    "nav-frontend-paneler": "^2.0.10",
```

Comment:
Oppdater denne gjerne for alle pakkene for den finnes slik at vi ungår undøvendige dependencies.

Kan gjøre med en find and replace all eller yarn upgrade-interactive --latest for så finne pakken

Reply:
aha, min skyld ;)